Automattic/wp-calypso
# Revert "Add instructional 'Fonts' section to customizer when fonts are only modifiable from 'Global Styles' (#54191)"

This reverts pull request [54191](https://github.com/Automattic/wp-calypso/pull/54191) (commit:  0e648d8d7e4f8747806e040dc6990961c96f523b)

#### Testing instructions

1. Host a calypso app locally: `cd wp-calypso; yarn && yarn start`
1. Host the editing toolkit plugin as well. `cd wp-calypso/apps/editing-toolkit; yarn dev --sync`
1. Make sure your hosts file is pointing wordpress.com and public-api.wordpress.com to your sandbox, as well as the specific site you're about to test with.
1. Make sure you have an active SSH connection to your sandbox
1. Perform all the following steps using your locally hosted calypso app.
1. Activate a theme which requires fonts to be modified from global styles (such as twenty twenty one)
1. Go to customizer
1. There should not be a 'fonts' section

#### Atomic Testing Instructions (This won't be deployed to atomic)

1. Create an atomic site
2. Make sure your hosts file is pointing to production servers
3. Download the zipped version of the editing toolkit plugin from [here](https://teamcity.a8c.com/viewLog.html?buildId=6417514&buildTypeId=calypso_WPComPlugins_EditorToolKit&tab=artifacts#).
4. Install/activate this plugin (You may need to deactivate the existing editing toolkit plugin first)
3. Activate a theme which requires fonts to be modified from global styles (such as twenty twenty one)
4. Go to customizer
5. There should not be a 'fonts' section
